tests/test_transformers.py

```diff
@@ -560,48 +609,7 @@ def test_temperature(self):
 
     @pytest.mark.nofbcode
     def test_resize_embeddings(self):
```

Comment:
tbh I think i would rather have the separate tests i think, and not try to DRY this...

Reply:
not sure what you mean - are you supporting current setup? or would prefer more specialized testing for each model

Reply:
I'd prefer specialized tests and not doing this refactor here. But I yield to you.

Reply:
i think this test right now is adequate for the functionality being tested - i.e., that an agent is able to handle additional special tokens to the dictionary. As alluded to in the PR description, this ideally would be a top level `TorchAgent` function, which would indeed only require a set of tests for TA specifically - including this test. 

So, for now, i think this CI is appropriate. but happy to add more testing once this becomes top level functionality